ivy/container/base.py

```diff
@@ -3685,6 +3685,7 @@ def _align_arrays(str_in):
                     and (self._ivy.is_native_array(v[0]) or isinstance(v[0], ivy.Array))
                 ):
                     rep = (
+                        v,
                         "list[{}]".format(len(v)),
```

Comment:
we always show `list` even if it is a `tuple`, we should show whichever is correct. Also, this just shows the full value as well as all of the stats about the `list` or `tuple`. We only want to print things differently if it's a named tuple I would say. This doesn't solve the problem. We don't want it to always show the entire list. If this contains 1000s of entries then we want to show the stats instead for example.